docs/examples/AccelerometerExample/src/AccelerometerExample.ls

```diff
@@ -59,10 +68,13 @@ package
             sprite.y = Math.clamp(sprite.y, 0, stage.stageHeight);
         }
 
-        protected function onAccelerometerData(x:Number, y:Number, z:Number) 
+        protected function onAccelerometerData(type:MobileSensorType, x:Number, y:Number, z:Number)
```

Comment:
It's a generic event that can get fired for other kinds of sensors, yes. And yes, maybe `onSensorTriple` might be more correct.